AboutPage.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/css/Nabvar.css">
+  <link rel="stylesheet" href="/css/burgerMenuWindows.css">
+  <link rel="stylesheet" href="/css/dinamicSection.css">
+  <link rel="stylesheet" href="/css/about.css">
+  <title>CapstoneFernando</title>
+</head>
+<body>
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
+        <ul class="nav-list"> 
+          <li>
+            <a href="AboutPage.html">About</a>
+          </li>
+          <li>
+            <a href="#Program">Program</a>
+          </li>
+          <li>
+            <a href="#Join">Join</a>
+          </li>
+          <li>
+          <li>
+            <a href="#Sponsor">Sponsor</a>
+          </li>
+          <li>
+            <a href="#News">News</a>
+          </li>
+            <a href="#" class="header_button" id="header_button">UC Campaigns</a>
+          </li>
+        </ul>
+      </header>
+
+  <main>
+    <section class="mobile-menu" id="mobileNave">
+      <ul class="nav-list2"> 
+        <img onclick="closeNave();" class="img-close" src="./images/Icon_X_.png" alt="">
```

Comment:
- [ ] Would you mind adding alt properties to your images? This simple step would greatly enhance the accessibility of your page for a wider range of users.